audience_manager/api.rst

```diff
@@ -0,0 +1,213 @@
+.. highlight:: js
+.. default-domain:: js
+
+Audience Manager JavaScript API
+===============================
+This API provides JavaScript access to multiple resources.
+
+Loading snippet
+---------------
+Add following snippet on a page to start using Audience Manager JS API.
```

Comment:
"Following snippet" suggests that below the paragraph the snippet is located while it's 3 sections below. It was confusing to me when I was reading it for the first time.